lua/neotest-dart/init.lua

```diff
@@ -198,7 +198,9 @@ setmetatable(adapter, {
           if client.name == 'dartls' and is_test_file then
             local originalOutline = client.handlers['dart/textDocument/publishOutline']
             client.handlers['dart/textDocument/publishOutline'] = function(_, data)
-              originalOutline(_, data)
+              if originalOutline ~= nil then
```

Comment:
It should be enough to have this:
```suggestion
              if originalOutline then
```